config/dispatcher/dispatcher.ex

```diff
@@ -15,10 +15,19 @@ defmodule Dispatcher do
   get "/sync/files/*path" do
     forward conn, path, "http://published-resource-producer/files/"
   end
+  
   match "/files/*path" do
     forward conn, path, "http://file/files/"
   end
 
+  match "/file-resources/*path" do
```

Comment:
I think typically we match the post to /files and have the rest go to resources. See this example:
```suggestion
  post "/files/*path" do
    Proxy.forward conn, path, "http://file/files/"
  end
  get "/files/*path" do
    Proxy.forward conn, path, "http://resource/files/"
  end
  patch "/files/*path" do
    Proxy.forward conn, path, "http://resource/files/"
  end

  delete "/files/*path" do
    Proxy.forward conn, path, "http://file/files/"
  end
```